modules/reports/client-react/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Route, NavLink } from 'react-router-dom';
+import { translate, TranslateFunction } from '@module/i18n-client-react';
+import { MenuItem } from '@module/look-client-react';
+import ClientModule from '@module/module-client-react';
+
+import Report from './containers/Report';
+import resources from './locales';
+
+const NavLinkWithI18n = translate('report')(({ t }: { t: TranslateFunction }) => (
+  <NavLink to="/report" className="nav-link" activeClassName="active">
+    {t('navLink')}
+  </NavLink>
+));
+
+export default new ClientModule({
```

Comment:
The Report `ClientModule` should have the same structure the server has. That means it should be divided into several modules: `excel`, `pdf`, and each of them provide its part of functionality. Currently all the logic is blended.